### ISSUE TITLE
Interim Report: Related Work

### ISSUE THREAD

**Issue by Pandakar** [cscw2015/maverick#2]
We need to reorganize this section and possibly relabel sections of it. Current sub-sections are:
- CSCW Use in Government and Politics
- CSCW in Meetings
- Group Dynamics

Need to highlight the history of the field as well as how previous research was conducted, and why.
Should also include current events regarding transparency in BC government.

**Comment by cscw2015** [comment]
Do you still think we should make points of the information that is already in there? I can do that (since I wrote half of it).

**Comment by Pandakar** [comment]
Yes, I think that we should reduce it to point form. It'll be easier that way to move ideas around and see what ideas fit into our related work, and it'll also allow us to pinpoint any specific points that don't fit into our project.

**Comment by cscw2015** [comment]
Hi everyone,

I went back and looked at all of the papers we cited in the Related Works section the first time around, and I think we should expand on the concepts in the papers Roshni found. Specifically, I think a 
section of our report should be on various technologies and why they were chosen by various governments. I have identified the following technologies that I think should be headings (in order):
History of CSCW Use in Government
- Cuparla (2000)
- e-government (~2002)
- GIS (2009) -- not sure if we should include this one
- open-source (2014)
- Electronic voting system (unknown year)
- UK government G-cloud (2015)
- Government 2.0 (current)
  I have attached a document that goes into more detail.
  [History of CSCW in Government.txt](https://github.com/cscw2015/maverick/files/29456/History.of.CSCW.in.Government.txt)
